base/common/CMakeLists.txt

```diff
@@ -49,6 +46,16 @@ add_custom_command(
     COMMAND ${CMAKE_COMMAND} -E create_symlink ${COMMONS_NET_JAR} lib/commons-net.jar
 )
 
+if(JAVA_VERSION GREATER 10)
+    add_custom_target(pki-java11plus-lib ALL
+        COMMENT "Creating links for library required in Java 11+")
+
+    add_custom_command(
+        TARGET pki-java11plus-lib
+        COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAKARTA_ACTIVATION_JAR} lib/jakarta.activation.jar
```

Comment:
Needs to be:

```
    COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAXB_API_JAR} lib/jaxb-api.jar
    COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAXB_IMPL_JAR} lib/jaxb-impl.jar
    COMMAND ${CMAKE_COMMAND} -E create_symlink ${JAKARTA_ACTIVATION_JAR} lib/jakarta.activation.jar
```

i.e., all three of them. Sorry I wasn't clear last time. :-)